### PR TITLE
Support excluding files or directories from analyze_project.

### PR DESCRIPTION
**Pull request by rchen152**
* Supports an --exclude flag to make it easy to exclude parts of
  a directory that are not pytype-safe (e.g.,
  `pytype subpackage --exclude subpackage/*_test.py`)
* Renames the filenames arg to inputs in preparation for adding
  it to the configuration file, since `filenames` is a bit of a
  misnomer when we allow directories as well.

This PR will have to be imported into Google and then re-exported.

### CODE REVIEW

File: pytype/tools/analyze_project/main.py [hunk 1]
```diff
@@ -59,10 +61,10 @@ def main():
   # Command line arguments overwrite file options.
   conf.populate_from(args)
   if not conf.pythonpath:
-    conf.pythonpath = environment.compute_pythonpath(args.filenames)
+    conf.pythonpath = environment.compute_pythonpath(args.inputs)
   logging.info('\n  '.join(['Configuration:'] + str(conf).split('\n')))
 
-  if not args.filenames:
+  if not args.inputs:
     parser.parser.print_usage()
```
Comment (martindemello) [inline pytype/tools/analyze_project/main.py]: while you're at it, add a print statement here saying "No input files" - just printing out the usage can be unhelpful, especially if we add inputs to the config file.

Comment (rchen152) [reply pytype/tools/analyze_project/main.py]: Done.

**Comment by rchen152** [comment]
This PR changed files that are tracked internally, so it should've gone through the Copybara PR workflow rather than being merged directly. (I noted that in the initial description.) I *think* to fix things now I can just make the same change internally and Copybara will figure everything out; let me try.

**Comment by martindemello** [comment]
ah, i missed that comment in the initial description, sorry.

**Pull request merged**